Sources/TestWebService/main.swift

```diff
@@ -23,6 +23,14 @@ struct TestWebService: Apodini.WebService {
         SwiftComponent()
         UserComponent(userId: _userId)
     }
+    
+    var configuration: Configuration {
+        OpenAPIConfiguration(
+            outputFormat: .json,
+            outputEndpoint: "oas",
+            swaggerUiEndpoint: "oas-ui",
+            title: "The great TestWebService - presented by Apodini")
```

Comment:
Very nitpicky: I would move the parenthesis to the next line.
```suggestion
            title: "The great TestWebService - presented by Apodini"
        )
```

Reply:
👍 09f5109